app/design/frontend/Magebit/learning/Magento_Catalog/templates/product/view/relatedproducts.phtml

```diff
@@ -1,33 +1,54 @@
 <?php
+/**
+ * @copyright Copyright (c) 2024 Magebit, Ltd. (https://magebit.com/)
+ * @author    Magebit <info@magebit.com>
+ * @license   MIT
+ */
+
+declare(strict_types=1);
+
+use Magebit\Learning\Block\Product\View\RelatedProducts;
+use Magento\Framework\Escaper;
+use Magento\Framework\Exception\NoSuchEntityException;
+
 /**
  * Template for displaying related products in a grid format.
- * @var \Magento\Catalog\Block\Product\View\Attributes $block
+ * @var RelatedProducts $block
+ * @var Escaper $escaper
  */
+?>
+<?php try {
+    if ($relatedProducts = $block->getRelatedProducts()): ?>
+        <?php if ($relatedProducts->getSize()): ?>
+            <div class="related-products">
+                <h2><?= $escaper->escapeHtml(__('Related Products')) ?></h2>
+                <hr>
+                <ul class="products-grid">
+                    <?php foreach ($relatedProducts as $product): ?>
+                        <li class="product-item">
+                            <a href="<?= $escaper->escapeUrl($block->getProductUrls($product)) ?>">
+                                <img src="<?= $escaper->escapeUrl($block->getProductImageUrl($product)) ?>"
+                                     alt="<?= $escaper->escapeHtml($block->getProductName($product)) ?>"/>
+                                <h3><?= $escaper->escapeHtml($block->getProductName($product)) ?></h3>
+                            </a>
+                            <div class="product-reviews">
+                                <?= $block->getReviewsSummaryHtml($product, 'short', true) ?>
```

Comment:
When you don't want to escape html, you can add <?= /* @noEscape */ $block->getReviewsSummaryHtml($product, 'short', true) ?>